packages/core/src/queries/user.ts

```diff
@@ -108,3 +108,15 @@ export const deleteUserById = async (id: string) => {
     throw new DeletionError(Users.table, id);
   }
 };
+
+export const deleteUserCustomDataById = async (id: string) => {
```

Comment:
What about `clearUserCustomData`?, and it seems that we already have some update functions?

Though the requset method is `DELETE`, it dosen't mean we have to name it as "delete" in our code.

Reply:
updated